jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -485,39 +487,38 @@
-        #         export FAIL_ON_GCP_RESOURCE_LEAK="false"
-        #         # Override GCE defaults.
-        #         export KUBE_GCE_ZONE="us-east1-b"
-        #         export MASTER_SIZE="n1-standard-4"
-        #         export NODE_SIZE="n1-standard-1"
-        #         export NODE_DISK_SIZE="50GB"
-        #         export NUM_NODES="100"
-        #         export REGISTER_MASTER="true"
-        #         # Reduce logs verbosity
-        #         export TEST_CLUSTER_LOG_LEVEL="--v=2"
-        #         # TODO: Remove when we figure out the reason for occasional failures #19048
-        #         export KUBELET_TEST_LOG_LEVEL="--v=4"
-        #         # Increase resync period to simulate production
-        #         export TEST_CLUSTER_RESYNC_PERIOD="--min-resync-period=12h"
-        #         # Increase delete collection parallelism
-        #         export TEST_CLUSTER_DELETE_COLLECTION_WORKERS="--delete-collection-workers=16"
-        #         export KUBE_NODE_OS_DISTRIBUTION="debian"
+        - 'gce-scalability-release-1.5':  # kubernetes-e2e-gce-scalability-release-1.5
+            description: 'Run scalability E2E tests on GCE from the release-1.5 branch.'
+            timeout: 120
+            cron-string: '@daily'
+            job-env: |
+                export JENKINS_PUBLISHED_VERSION="ci/latest-1.5"
+                export E2E_NAME="e2e-scalability-1-5"
+                export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Performance\] \
+                                         --gather-resource-usage=true \
+                                         --gather-metrics-at-teardown=true \
+                                         --gather-logs-sizes=true \
+                                         --output-print-type=json"
+                # Use the 1.1 project for now, since it has quota.
+                # TODO: create a project k8s-e2e-gce-scalability-release and move this test there
+                export PROJECT="k8s-e2e-gce-scalability-1-1"
+                export FAIL_ON_GCP_RESOURCE_LEAK="false"
+                # Override GCE defaults.
+                export KUBE_GCE_ZONE="us-east1-b"
+                export MASTER_SIZE="n1-standard-4"
+                export NODE_SIZE="n1-standard-1"
+                export NODE_DISK_SIZE="50GB"
+                export NUM_NODES="100"
+                export REGISTER_MASTER="true"
+                # Reduce logs verbosity
+                export TEST_CLUSTER_LOG_LEVEL="--v=2"
+                # TODO: Remove when we figure out the reason for occasional failures #19048
+                export KUBELET_TEST_LOG_LEVEL="--v=4"
+                # Increase resync period to simulate production
+                export TEST_CLUSTER_RESYNC_PERIOD="--min-resync-period=12h"
+                # Increase delete collection parallelism
+                export TEST_CLUSTER_DELETE_COLLECTION_WORKERS="--delete-collection-workers=16"
+                export KUBE_NODE_OS_DISTRIBUTION="debian"
```

Comment:
To clarify - this is consistent with other suites in 1.5 branch now. The question is whether this is what we want.